### PR TITLE
mariadodoll.json

### PR DESCRIPTION
**Pull request by mariadodoll**
Added a new pin and now creating a pull request for Hector@githubteacher. Closes #3

### CODE REVIEW

File: _pins/mariadodoll.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+githubHandle: mariadodoll
+latitude: 28.375144
+longitude:-81.379234
```
Comment (crichID) [inline _pins/mariadodoll.json]: @mariadodoll If you take a look at why the build is failing, it says `could not find expected ':' while scanning a simple key at line 4 column 1`. In plain english, you need a space after the `:` on line 4 and then it should work!

File: _pins/mariadodoll.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+---
+Adding a pin
```
Comment (crichID) [inline _pins/mariadodoll.json]: @mariadodoll you should remove this extra line!

**Comment by merxn4x** [comment]
I am brand new to this and coding. I am not sure what you mean?

**Comment by crichID** [comment]
> I am brand new to this and coding. I am not sure what you mean?

@merxn4x great question! In this case, I was specifically providing feedback to @mariadodoll on her pull request. There are some changes I would like her to make that will allow her to finish the process of merging her pull request!

**Comment by AumDove** [comment]
You can do this! :wink: :ok_hand:
